packages/components/src/components/tooltip/tooltip.css

```diff
@@ -0,0 +1,164 @@
+host*,
```

Comment:
Is this a valid selector?

Reply:
Yes I think it was something like selecting all child elements of the parent element(host)

Reply:
I tested it and it seems `host*` is not selecting any elements 🤪 (you can see that in devtools)

You can either use:
- `:host` for selecting the host, in this case `scale-tooltip`
- `:host(selector)` to selecting the `scale-tooltip` with some extra selectors like a class, or
- `*` to select all elements inside the shadow root, _excluding_ the host

Is the `box-sizing: inherit;` declaration actually needed? if so, in which elements?

Reply:
I needed it before but it seems to do nothing in the actual version, so I removed it for now.😅

Thanks for the detailed explanation😁